mmi-cookbooks/metrics-repose
# 0.1.1 version

# What 
Update wrapper to version 0.1.1.  Includes updates to the logging format strings to include additional HTTP request and response headers.  Add an include for the repose::install recipe to make sure that repose is installed and dirs like /etc/repose exist before we try putting files there.

# Why
Working on logging improvements as we roll out to prod so that we have similar logs to what we saw with version 6.
We are upgrading to logstash 1.5.5 with filebeat streaming logs to it.  With the upgrade to repose 7.3.5.0, log4j is also upgraded to log4j2 - which doesn't have the JSON layout format that we have been using in log4j with repose.  (reference - https://github.com/logstash/log4j-jsonevent-layout/pull/56)

# How to Test
Deploy to staging and check that /etc/repose/slf4j-http-logging.cfg.xml is properly updated.
